arch/arm64/boot/dts/qcom/sdm636-bbry-luna.dts

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2025, Paul Sajna <hello@paulsajna.com>
+ */
+
+/dts-v1/;
+
+#include "sdm636.dtsi"
+#include "sdm660-bbry-common.dtsi"
```

Comment:
I think first should come include `-common`, then `sdm636`

Because common includes `sdm660.dtsi`, and `sdm636.dtsi` also includes `sdm660.dtsi` and then applies overrides on top

Reply:
Though maybe it doesn't matter, because https://github.com/sdm660-mainline/linux/blob/qcom-sdm660-6.15.y/arch/arm64/boot/dts/qcom/sdm636.dtsi so far only overrides compatible string on `&adreno_gpu` and dtc combines them all together while compiling anyways :thinking:

Reply:
oop, compiler didn't like that

Reply:
I think maybe I should pull 660 out of common?

Reply:
alright, whatever works..